openupgrade_scripts/scripts/product/18.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2025 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+xmlid_renames = [
+    ("product.group_discount_per_so_line", "sale.group_discount_per_so_line"),
+]
+
+column_creates = [
+    ("product.attribute", "active", "boolean", "TRUE"),
+    ("product.attribute.value", "active", "boolean", "TRUE"),
+    ("product.pricelist.item", "display_applied_on", "varchar", "'1_product'"),
+]
+
+
+def rename_pos_models(env):
+    """
+    pos.combo and pos.combo.line have been moved to product from point_of_sale and
+    renamed to product.combo and product.combo.item respectively
+    """
+    if not openupgrade.table_exists(
+        env.cr, "pos_combo"
+    ) or not openupgrade.table_exists(env.cr, "pos_combo_line"):
+        return
```

Comment:
```suggestion
    if not openupgrade.table_exists(
        env.cr, "pos_combo"
    ):
        return
```
If they have combo, they will have combo line. And if they have combo line, they will have combo.

Reply:
done